frontend/src/components/core/Sidebar/Sidebar.tsx

```diff
@@ -92,6 +115,10 @@ class Sidebar extends PureComponent<SidebarProps, State> {
     this.mediumBreakpointPx = Sidebar.calculateMaxBreakpoint(
       this.props.theme.breakpoints.md
     )
+
+    const cachedSidebarNavExpandedPreference =
+      Sidebar.getSidebarNavPreference()
+
```

Comment:
This can then be removed because we are removing the `setState` call below that uses it

Reply:
We might still need it, per [my comment above](https://github.com/streamlit/streamlit/pull/6050#discussion_r1121723933), but let me know if you have a better way to address this, and I'll get it fixed!